catroid/build.gradle

```diff
@@ -86,6 +86,7 @@ android {
             buildConfigField "boolean", "FEATURE_RASPI_ENABLED", "true"
             buildConfigField "boolean", "FEATURE_POCKETMUSIC_ENABLED", "true"
             buildConfigField "boolean", "FEATURE_CAST_ENABLED", "true"
+            buildConfigField "boolean", "FEATURE_CLOUD_MESSAGING", "false"
```

Comment:
To be consistent, please rename to `FEATURE_CLOUD_MESSAGING_ENABLED`.
Why using `false`?